lib/catList.dart

```diff
@@ -0,0 +1,8 @@
+import 'package:flutter/material.dart';
+
```

Comment:
Try to add some structure to the project.
move custom widget to lib/ui/widgets/ package.

Reply:
fixed